src/views/clusteroverview/overview/components/inventory-card/utils/ResourcesSection.tsx

```diff
@@ -7,43 +7,59 @@ import { K8sResourceCommon, WatchK8sResults } from '@openshift-console/dynamic-p
 import { ResourceInventoryItem } from '@openshift-console/dynamic-plugin-sdk-internal';
 import { Stack, StackItem } from '@patternfly/react-core';
 
-import { InventoryCardResources } from './types';
-import { getOSImagesNS, getTemplates } from './utils';
+import { getAllowedResourceData } from '../../../utils/utils';
+
+// import { InventoryCardResources } from './types';
+import { getOSImagesNS } from './utils';
 
 import './ResourcesSection.scss';
 
 export type ResourcesSectionProps = {
-  resources?: WatchK8sResults<InventoryCardResources>;
+  resources?: WatchK8sResults<{
+    [key: string]: K8sResourceCommon[];
+  }>;
 };
 
 const ResourcesSection: React.FC<ResourcesSectionProps> = ({ resources }) => {
-  const templates = React.useMemo(() => getTemplates(resources), [resources]);
+  const templates = React.useMemo(
+    () => getAllowedResourceData(resources, TemplateModel),
+    [resources],
+  );
+  const vms = React.useMemo(
+    () => getAllowedResourceData(resources, VirtualMachineModel),
+    [resources],
+  );
+  const nads = React.useMemo(
+    () => getAllowedResourceData(resources, NetworkAttachmentDefinitionModel),
+    [resources],
+  );
   const dataSourceNS = React.useMemo(() => getOSImagesNS(), []);
 
+  console.log('🚀 ~ file: ResourcesSection.tsx ~ line 46 ~ vms?.loadError', vms?.loadError);
```

Comment:
@glekner the automatic tests should have failed the PR on this line, is seepSource broken ?